multus/multus.go

```diff
@@ -263,10 +263,12 @@ func cmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) (cn
 	}
 
 	//set the network status annotation in apiserver, only in case Multus as kubeconfig
-	if n.Kubeconfig != "" && kc.Podnamespace != "kube-system" {
-		err = k8s.SetNetworkStatus(kc, netStatus)
-		if err != nil {
-			return nil, fmt.Errorf("Multus: Err set the networks status: %v", err)
+	if n.Kubeconfig != "" && kc != nil {
```

Comment:
How about to change multus.go:249 as well as this for graceful bailout?
It also has `if n.Kubeconfig != "" && kc.Podnamespace != "kube-system" {`.

Reply:
Oh yes, I missed it out. My bad :)